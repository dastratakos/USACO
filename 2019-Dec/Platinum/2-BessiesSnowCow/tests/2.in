100 200
41 77
35 77
14 77
82 14
88 82
6 82
98 6
61 98
71 6
87 71
5 87
83 5
34 83
31 5
81 31
38 31
26 31
62 38
70 81
60 38
12 70
27 26
22 27
66 12
56 66
52 56
30 56
18 56
64 30
43 52
76 18
90 76
55 76
23 90
4 55
28 4
67 28
59 67
42 4
91 42
74 91
19 74
48 59
58 48
25 74
54 48
7 58
78 7
29 78
15 29
1 15
65 1
44 65
68 44
24 68
73 24
20 73
75 73
32 20
95 32
49 32
17 95
99 17
93 49
51 99
8 99
80 8
53 80
94 53
45 94
72 94
2 72
50 72
85 50
10 85
40 10
9 40
92 85
97 9
46 97
13 46
21 13
96 97
86 96
79 96
100 96
89 100
84 79
11 84
37 11
33 37
36 33
63 11
16 63
47 16
39 47
69 39
57 47
3 57
2 46
2 19
2 87
1 41 9122
1 83 61064
2 39
2 66
2 89
1 80 162
1 36 78026
2 7
1 22 79910
1 33 81797
2 32
1 19 72316
2 96
1 97 42429
1 27 42088
1 78 19111
1 25 94996
1 26 74266
2 9
2 34
1 17 76780
1 83 45169
2 38
1 39 79070
2 55
2 67
1 75 24314
2 50
1 47 32931
1 50 38285
2 20
2 73
1 6 31685
2 14
2 100
2 43
1 36 14385
1 91 17371
2 55
2 9
2 15
1 74 72022
1 23 2023
2 90
1 16 37448
1 8 20933
1 40 10168
1 9 16422
2 92
1 83 54905
1 72 30463
1 43 38285
1 35 92657
1 24 60224
1 64 60083
1 71 93609
1 7 39177
2 58
2 94
1 25 93075
1 95 79818
1 67 57141
2 29
1 27 59049
2 76
1 22 90028
2 40
2 42
1 61 46696
2 3
2 73
2 95
2 75
2 90
1 80 39842
2 45
1 94 6561
1 68 89024
1 84 24019
1 17 78874
2 80
2 48
2 88
2 94
1 7 70352
1 10 70815
2 64
2 99
1 23 77691
2 3
2 9
2 80
1 71 25792
2 22
2 67
1 17 77837
2 32
1 31 83233
1 36 96456
2 20
1 100 23776
2 90
2 38
2 56
1 2 5867
1 73 88786
2 92
1 73 99574
1 69 39108
2 40
1 12 17852
2 32
1 93 114
1 27 13062
1 7 98258
2 92
2 15
1 93 69984
1 21 17947
2 2
2 8
1 100 15130
2 38
1 67 60469
1 4 1708
1 97 57689
2 97
1 25 61930
1 94 38539
1 48 31734
2 39
1 64 81273
1 28 64443
2 90
1 87 32041
1 2 45977
2 42
2 32
2 74
2 44
2 48
2 21
2 25
2 40
2 16
2 81
2 18
1 79 70277
1 41 21238
2 11
1 43 3464
2 48
2 21
1 44 82914
1 25 69718
1 88 31146
1 89 97285
2 21
1 27 37917
2 29
1 64 46152
2 15
1 11 77614
2 5
2 42
2 68
1 86 84927
1 98 26497
1 26 64013
1 58 32577
1 88 45974
2 66
1 17 54385
2 25
2 61
1 3 91312
1 35 81132
2 17
1 42 77537
2 21
1 40 12509
2 73
1 17 38616
2 23
2 26
2 75
1 5 93389
1 31 18526
1 1 28035
2 77
1 95 39834
2 3
2 76
2 41
1 28 56199
2 87
1 85 19721
