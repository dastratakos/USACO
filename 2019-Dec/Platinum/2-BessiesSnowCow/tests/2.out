0
0
0
0
3
0
3
36
20
46
2
3
46
56
80
128
129
9
8
1
55
98
75
3
4
139
263
186
140
244
25
12
326
319
2
7
6
446
185
5
430
5
486
18
358
470
9
233
537
540
10
15
190
13
444
624
14
344
14
584
21
435
49
20
56
682
37
698
520
21
14
491
140
301
387
261
522
21
22
535
536
205
57
769
365
16
18
764
23
826
12
52
6
61
32
532
21
225
